autoPyTorch/pipeline/components/preprocessing/tabular_preprocessing/feature_preprocessing/LibLinearSVCPreprocessor.py

```diff
@@ -0,0 +1,141 @@
+from typing import Any, Dict, Optional
+
+from ConfigSpace.configuration_space import ConfigurationSpace
+from ConfigSpace.forbidden import (
+    ForbiddenAndConjunction,
+    ForbiddenEqualsClause,
+)
+from ConfigSpace.hyperparameters import (
+    CategoricalHyperparameter,
+    UniformFloatHyperparameter,
+    UniformIntegerHyperparameter
+)
+
+import numpy as np
+
+from sklearn.base import BaseEstimator
+from sklearn.feature_selection import SelectFromModel
+from sklearn.svm import LinearSVC
+
+
+from autoPyTorch.datasets.base_dataset import BaseDatasetPropertiesType
+from autoPyTorch.pipeline.components.preprocessing.tabular_preprocessing.feature_preprocessing. \
+    base_feature_preprocessor import autoPyTorchFeaturePreprocessingComponent
+from autoPyTorch.utils.common import HyperparameterSearchSpace, add_hyperparameter, get_hyperparameter
+
+
+class LibLinearSVCPreprocessor(autoPyTorchFeaturePreprocessingComponent):
+    """
+    Selects features based on importance weights using svm classifier
+    """
+    def __init__(self, dual: bool = False, penalty: str = "l1",
+                 loss: str = "squared_hinge", tol: float = 1e-4,
+                 C: float = 1, multi_class: str = "ovr",
+                 intercept_scaling: int = 1, fit_intercept: bool = True,
```

Comment:
[REF](https://scikit-learn.org/stable/modules/generated/sklearn.svm.LinearSVC.html)
```suggestion
                 intercept_scaling: float = 1, fit_intercept: bool = True,
```

Reply:
thanks for this